### PR TITLE
added sub-menu-arrow('>') and remove kebab menu in rootComment

### PR DESCRIPTION
**Pull request by Rash419**
Signed-off-by: Rash419 <rasheshpadia419@gmail.com>
Change-Id: I11e526fb6c2ad209c9231d75a655a6287a4f4bb5

* Target version: master 

### Summary
Screenshot from 2020-12-09 13-16-33

### CODE REVIEW

File: loleaflet/src/control/Control.JSDialogBuilder.js [hunk 1]
```diff
@@ -1844,11 +1844,16 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 			$(childContainer).hide();
 
-			if (builder.wizard)
+			if (builder.wizard) {
+				L.DomUtil.remove($(container).find('td')[3]);
```
Comment (lpranam) [inline loleaflet/src/control/Control.JSDialogBuilder.js:1848]: use `$(container).find(".loleaflet-annotation-menubar")[0]` instead

File: loleaflet/src/control/Control.JSDialogBuilder.js [hunk 2]
```diff
@@ -1844,11 +1844,16 @@ L.Control.JSDialogBuilder = L.Control.extend({
...
+				var arrowSpan = L.DomUtil.create('span','sub-menu-arrow',$(container).find('div')[0]);
```
Comment (lpranam) [inline loleaflet/src/control/Control.JSDialogBuilder.js:1849]: use `$(container).find('.loleaflet-annotation-content-wrapper')[0]` instead

**Review by lpranam — CHANGES_REQUESTED** [review]
It is always preferred to use better identifiers like class names or ids. Because in future if someone adds another element in the comment, your logic would fail and delete an incorrect element or who knows if layout breaks entirely.

File: loleaflet/src/control/Control.JSDialogBuilder.js (hunk 1 above)
Comment (Rash419) [reply loleaflet/src/control/Control.JSDialogBuilder.js:1848]: I got it and thanks, I will take care of it in future

File: loleaflet/src/control/Control.JSDialogBuilder.js (hunk 2 above)
Comment (Rash419) [reply loleaflet/src/control/Control.JSDialogBuilder.js]: Thanks :+1:

**Comment by pedropintosilva** [comment]
Hi @Rash419 and thanks for this! I have checked and I think it's pretty cool. 

I like it :heart:  Plus if we are in presence of a thread there are many other advantages that come for free by removing the 3 dots up top and only showing it when entering the thread (1. it avoids people commenting or asking stuff that was already answered or discussed in the thread; plus 2. It can be a nice distinction between simple comment and comment thread). **So great job ; )**

In addition we just need to be sure **to avoid removing the 3 dots when in presence of a simple comment**. (currently it  removes and adds the chevron even for non threaded comments)

Meanwhile I also have added a class that it will probably make this ^ easier. So I'm thinking to **merge this right now** and then, if you feel like, it would be cool to have another pull request to solve that. Does that sound ok?

**Review by pedropintosilva — APPROVED** [review]

**Review by pedropintosilva — APPROVED** [review]

**Comment by Rash419** [comment]
> Hi @Rash419 and thanks for this! I have checked and I think it's pretty cool. 
> 
> I like it :heart:  Plus if we are in presence of a thread there are many other advantages that come for free by removing the 3 dots up top and only showing it when entering the thread (1. it avoids people commenting or asking stuff that was already answered or discussed in the thread; plus 2. It can be a nice distinction between simple comment and comment thread). **So great job ; )**
> 
> In addition we just need to be sure **to avoid removing the 3 dots when in presence of a simple comment**. (currently it  removes and adds the chevron even for non threaded comments)
> 
> Meanwhile I also have added a class that it will probably make this ^ easier. So I'm thinking to **merge this right now** and then, if you feel like, it would be cool to have another pull request to solve that. Does that sound ok?



Your welcome but it was not my idea , @lpranam assigned me to implement this . It was his idea👍.

**Comment by lpranam** [comment]
@Rash419 before we do that I have a couple of more things lined up as an improvement maybe we can revisit this later. It would require a bit more work so can keep it at last for now.

**Comment by Rash419** [comment]
> @Rash419 before we do that I have a couple of more things lined up as an improvement maybe we can revisit this later. It would require a bit more work so can keep it at last for now.

Ok

**Comment by pedropintosilva** [comment]
@lpranam : sure, let's follow your lead, and thanks for helping out @Rash419

**Pull request merged**